coins.json

```diff
@@ -1052,7 +1052,7 @@
         "name": "FIO",
         "symbol": "FIO",
         "decimals": 9,
-        "blockchain": "EOS",
+        "blockchain": "FIO",
```

Comment:
I would prefer to use blockatlas type as `EOS`, because we gonna integrate EOS in the future, and FIO is a fork of EOS

Reply:
that was my first thought, some work flow in FIO would be different with EOS

Reply:
that works 👍